src/vivarium_nih_us_cvd/components/causes/causes.py

```diff
@@ -0,0 +1,152 @@
+from importlib import import_module
+from typing import Any, Callable, Dict, List, Union
+
+import pandas as pd
+from vivarium import ConfigTree
+from vivarium.framework.engine import Builder
+from vivarium_public_health.disease import (
+    BaseDiseaseState,
+    DiseaseModel,
+    DiseaseState,
+    RecoveredState,
+    SusceptibleState,
+    TransientDiseaseState,
+)
+from vivarium_public_health.utilities import TargetString
+
+from vivarium_nih_us_cvd.constants.paths import CAUSE_RISK_CONFIG
+
+
+class Causes:
+    # todo could modify the component configuration parser plugin

```

Comment:
Any Jira needed for this?

Reply:
Yes, good reminder. I'll create a ticket